src/analysis/souffle/examples/multimic/multimic_micowner_policy.arcs

```diff
@@ -0,0 +1,30 @@
+// Policy for multimic as provided on the user’s mic
+
+schema ASRData { }
+
+@intendedPurpose('
+  Allow multimic ASR.
+
+  ALLOWED EGRESSES: ASR.
+  ALLOWED USAGES: Text to speech, demos.
+  ALLOWED POPULATIONS: Production builds.
+')
+// The priority order in which principal policies should be applied.
+// Lower numbers mean higher trust. Any principal not given a priority shall
+// be considered at the lowest priority. The number could easily be replaced
+// with an enum if these should be given names.
+//SPECULATIVE:
+//@PrincipalPriority(“UserA”, 0)
+//@PrincipalPriority(“SpeechToTextSoftwareVendor”, 5)
+//@PrincipalPriority(“UserB”, 50)
+//@PrincipalPriority(“UserC”, 50)
+@egressType("Disk")
+policy UserASmartMicOwnerPolicy {
+  // textTranscript is a hypothetical label describing text derived from user
+  // speech instead of the raw audio itself.
+  @allowedUsage(label: 'textTranscript', usageType: '*')
+  @allowedRetention(medium: 'Ram', encryption: false)
+  @allowedRetention(medium: 'Disk', encryption: false)
+  from ASRData access { }
```

Comment:
Shouldn't this be from `AudioData` instead of `ASRData`? Essentially, don't you want to put restrictions on how raw audio stream is treated?

Reply:
Fixed.